build.sbt

```diff
@@ -84,14 +84,31 @@ lazy val zioConfigRefined =
     )
     .dependsOn(zioConfig % "compile->compile;test->test")
 
+lazy val runAllExamples = taskKey[Unit]("Run all main classes in examples module")
+
 lazy val examples = module("zio-config-examples", "examples")
   .settings(
     skip in publish := true,
     fork := true,
     libraryDependencies ++= Seq(
       "eu.timepit"     %% "refined"  % refinedVersion,
       "com.propensive" %% "magnolia" % magnoliaVersion
-    )
+    ),
+    runAllExamples :=
+      Def
+        .taskDyn({
+          val c    = (discoveredMainClasses in Compile).value
+          val runs = (runMain in Compile)
+
+          val x = c.map(cc => {
+            Def.task {
+              runs.toTask(s" ${cc}").value
+            }
+          })
+
+          Def.sequential(x)
+        })
+        .value
```

Comment:
Mainly intended for contributors.
We have 21 examples now, each one of them has an assertion to tell the user what's the output looking like,
After making changes, if someone needs to run all examples with in sbt shell, 

```
sbt runAllTasks
```
If any of the assertions fail, the task stops